c2corg_ui/static/js/user.js

```diff
@@ -100,4 +100,12 @@ app.UserController.prototype.logout = function() {
 };
 
 
+/**
+ * @export
+ */
+app.UserController.prototype.hasEditRightsOuting = function(users) {
+  this.auth.hasEditRightsOuting(users);
```

Comment:
See my other comment below: can't the check be done directly in the ``protected-url-btn`` directive along with the "is auth" check?